spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -1658,16 +1658,37 @@ def _get_phy_data(recording, sorting, compute_pc_features, compute_amplitudes,
     else:
         templates_ind = np.tile(np.arange(recording.get_num_channels()), (len(sorting.get_unit_ids()), 1))
 
+    # Reorder template with amplitude for phy
+    templates, templates_ind = _template_descending_order(recording, sorting, templates, templates_ind)
+
     # similar_templates.npy - [nTemplates, nTemplates] single
     similar_templates = _compute_templates_similarity(templates, templates_ind)
 
     # spike_templates.npy - [nSpikes, ] uint32
     spike_templates = spike_clusters
-
+    
     return spike_times, spike_clusters, amplitudes, channel_map, pc_features, pc_feature_ind, \
            spike_templates, templates, templates_ind, similar_templates, channel_map_si, channel_groups, positions
 
 
+def _template_descending_order(recording, sorting, templates, templates_ind):
```

Comment:
Why do you need `sorting` as argument?

Reply:
you are right, i was a bit too fast i when I create the function